spec/features/admin_claim_check_spec.rb

```diff
@@ -93,12 +93,17 @@
     context "When the claimant has not completed GOV.UK Verify" do
       let!(:claim_without_identity_confirmation) { create(:claim, :unverified) }
 
-      scenario "the service operator should be told the identity hasn't been confirmed" do
+      scenario "the service operator is told the identity hasn't been confirmed and can approve the claim" do
         click_on "View claims"
         find("a[href='#{admin_claim_path(claim_without_identity_confirmation)}']").click
 
-        expect(page).to have_content("Identity has not been confirmed")
-        expect(page).to have_field("Approve", disabled: true)
+        expect(page).to have_content("The claimant did not complete GOV.UK Verify")
```

Comment:
Is it worth checking for the presence of the correct school phone number here too?

Reply:
👍